versioned_docs/version-v2/guides/module-bundling.md

```diff
@@ -44,15 +44,13 @@ In this case, the compiler will emit an error that looks like this:
 
 The solution is to move any shared functions or classes to a different `.ts` file, like this:
 
```

Comment:
I can't comment on the line (too far up in the diff), but can we also do this file's equivalent [of this change?](https://github.com/ionic-team/stencil-site/pull/1194/files#diff-e7f3c4e8d646e467acaadb71ac7e495ae4dd952bfb6b5db615d1646c4a68a4ffR20)